src/deepqmc/wf/base.py

```diff
@@ -1,14 +1,25 @@
+from typing import Sequence
+
 import haiku as hk
 import jax
+import jax.numpy as jnp
 from jax.tree_util import tree_map, tree_reduce
 
+from ..types import PhysicalConfiguration
+
 __all__ = ['state_callback']
 
 
 def init_wf_params(rng, hamil, ansatz):
-    rng_hamil, rng_params = jax.random.split(rng)
-    r = hamil.init_sample(rng_hamil, 1)[0]
-    params, _ = ansatz.init(rng_params, r)
+    rng_R, rng_r, rng_params = jax.random.split(rng, 3)
+    phys_conf = PhysicalConfiguration(
```

Comment:
why do you need a `PhysicalConfiguration` to initialize the NN parameters?

Reply:
[haiku](https://dm-haiku.readthedocs.io/en/latest/notebooks/basics.html#A-first-example-with-hk.transform) needs a dummy input with the correct PyTree structure and array shapes, to initialize the model parameters.